src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -196,16 +193,34 @@ internal CommitManager(ITextView textView, RecentItemsManager recentItemsManager
             {
                 // We are on the wrong thread.
                 FatalError.ReportWithoutCrash(new InvalidOperationException("Subject buffer did not provide Edit Access"));
-                return AsyncCompletionData.CommitBehavior.None;
+                return new AsyncCompletionData.CommitResult(isHandled: true, AsyncCompletionData.CommitBehavior.None);
```

Comment:
note that this will prevent anyone else from attempting to commit

Reply:
Yes, I hope this is an exception scenario. I believe that we should be guaranteed to have an edit access and no edit in progress. If so, we need to yell/report that this assumption is not met. 

However, if this is not an exceptional situation, we should just agree about this and consider it from a different view.

@AmadeusW , please let me know your thoughts on this.